hassio/src/addon-store/hassio-addon-repository.ts

```diff
@@ -91,6 +95,23 @@ class HassioAddonRepositoryEl extends LitElement {
     navigate(this, `/hassio/addon/${ev.currentTarget.addon.slug}`);
   }
 
+  private fuzzySearchAndSort(addons, filter) {
```

Comment:
This function is in the code twice, can we reuse it instead of copy/paste?

Reply:
This is called from the render method, we also re-render when `hass` changes. We should wrap this in `memoize-one`, so that a `hass` change won't re-calculate it all. 

```suggestion
  private fuzzySearchAndSort = memoizeOne((addons, filter) => {
```

Reply:
If you extract the method into something like `filterAddons` like Bram suggests, this will become `memoizeOne(filterAddons)`